test/unittests/commands/xml_validate/xml_fixtures.py

```diff
@@ -0,0 +1,362 @@
+                    "status": "active",
+                    "type": "polygon",
+                    "lineWidth": 5,
+                    "points": [{"x": 0.4, "y": 0.6},
+                    {"x": 0.5, "y": 0.9},
+                    {"x": 0.8, "y": 0.9},
+                    {"x": 0.7, "y": 0.6}]
+                    }
+                </geometry>
+            </geometry-prop>
+            <text-prop name="http://api.knora.org/ontology/knora-api/v2#hasComment">
+                <text encoding="xml">
+                    This is a polygon-formed region of interest of an image. It is also displayed as Annotation.
+                </text>
+            </text-prop>
+        </region>
+    </knora>
+    """)
+
+
+@pytest.fixture
+def boolean_value_corr() -> etree._Element:
+    return etree.fromstring("""
+        <boolean-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testBoolean">
+            <boolean>true</boolean>
+        </boolean-prop>
+    """)
+
+
+@pytest.fixture
+def color_value_corr() -> etree._Element:
+    return etree.fromstring("""
+        <color-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testColor">
+            <color>#00ff00</color>
+        </color-prop>
+        """)
+
+
+@pytest.fixture
+def color_value_corr_several() -> etree._Element:
+    return etree.fromstring("""
+        <color-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testColor">
+            <color>#00ff00</color>
+            <color>#00ff11</color>
+        </color-prop>
+        """)
+
+
+@pytest.fixture
+def date_value_corr() -> etree._Element:
+    return etree.fromstring("""
+        <date-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testSubDate1">
+            <date>JULIAN:BCE:0700:BCE:0600</date>
+        </date-prop>
+        """)
+
+
+@pytest.fixture
+def date_value_corr_several() -> etree._Element:
+    return etree.fromstring("""
+        <date-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testSubDate1">
+            <date>JULIAN:BCE:0700:BCE:0600</date>
+            <date>ISLAMIC:BCE:0700:BCE:0600</date>
+        </date-prop>
+        """)
+
+
+@pytest.fixture
+def decimal_value_corr() -> etree._Element:
+    return etree.fromstring("""
+        <decimal-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testDecimalSimpleText">
+            <decimal>2.71</decimal>
+        </decimal-prop>
+    """)
+
+
+@pytest.fixture
+def decimal_value_corr_several() -> etree._Element:
+    return etree.fromstring("""
+        <decimal-prop name="http://0.0.0.0:3333/ontology/9999/onto/v2#testDecimalSimpleText">
+            <decimal>1.0</decimal>
+            <decimal>2.0</decimal>
+        </decimal-prop>
+    """)
+
+
+@pytest.fixture
+def geometry_value_corr() -> etree._Element:
+    return etree.fromstring("""
+        <geometry-prop name="hasGeometry">
+            <geometry>
+                {
+                    "status": "active",
+                    "type": "polygon",
+                    "lineColor": "#ff1100",
```

Comment:
we should not use `lineColor` anywhere, see https://github.com/dasch-swiss/dsp-tools/pull/1166

Reply:
I missed that thanks!